app/assets/javascripts/application.js

```diff
@@ -1,9 +1,3 @@
-// This is a manifest file that'll be compiled into including all the files listed below.
-// Add new JavaScript/Coffee code in separate files in this directory and they'll automatically
-// be included in the compiled file accessible from http://example.com/assets/application.js
-// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
-// the compiled file.
-//
 //= require jquery3
 //= require jquery_ujs
 //= require clipboard
```

Comment:
Since the `vendor/assets/javascripts/webauthn-json.js` was moved into vendor, we need to require it here using.

```
//= require webauthn-json

```

Reply:
oops, fixing that